MH4U Dex/HomeViewController.m

```diff
@@ -0,0 +1,37 @@
+//
+//  HomeViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/5/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "HomeViewController.h"
+
+#import "MonsterEncyclopediaViewController.h"
+#import "ItemEncyclopediaTableViewController.h"
+#import "RegionEncyclopediaCollectionViewController.h"
+
+@interface HomeViewController ()
+
+@end
+
+@implementation HomeViewController
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
+{
+    if ([segue.identifier isEqualToString:@"showMonsterEncyclopedia"]) {
```

Comment:
It would probably be good to have constants for these identifiers—perhaps generated from the storyboard using a tool like https://github.com/square/objc-codegenutils#objc-identifierconstants.